src/server/main_service.cc

```diff
@@ -1211,6 +1191,9 @@ void Service::DispatchCommand(CmdArgList args, facade::ConnectionContext* cntx)
     if (stored_cmd.Cid()->IsWriteOnly()) {
       dfly_cntx->conn_state.exec_info.is_write = true;
     }
+    if (cid->name() == "CLIENT") {
```

Comment:
@dranikpg  I disabled all `CLIENT` commands from being squashed

Reply:
Maybe it'd be better to stop squashing inside multi_command_squasher, so we don't pollute ExecInfo with a flag